langchain4j-agentic/src/test/java/dev/langchain4j/agentic/DeclarativeAgentIT.java

```diff
@@ -83,6 +90,35 @@ void declarative_single_agent_tests() {
         assertThat(story).isNotBlank();
     }
 
+    @Test
+    void declarative_streaming_agent_tests() throws Exception {
+        CreativeWriterForStreaming creativeWriter = AgenticServices.agentBuilder(CreativeWriterForStreaming.class)
+                .streamingChatModel(streamingBaseModel())
```

Comment:
In the declarative API all configurations, including the `ChatModel`, are provided via annotations. You should add something similar also for the `StreamingChatModel`.